compiler/include/symbol.h

```diff
@@ -383,6 +393,12 @@ class FnSymbol : public Symbol {
   bool            isMethod()                                   const;
   bool            isPrimaryMethod()                            const;
   bool            isSecondaryMethod()                          const;
+  bool            isIterator()                                 const;
```

Comment:
Yes, that is what I was suggesting.  Similar to what happened when isMethod() and company were added.